docs/source/guides/build.rst

```diff
@@ -0,0 +1,218 @@
+| TESTING      |      ON/OFF     | ON      | Enables or disables build of the tests                                 |
++--------------+                 +---------+------------------------------------------------------------------------+
+| BENCHMARKING |                 | OFF     | Enables or disables build of the benchmarks                            |
++--------------+                 +---------+------------------------------------------------------------------------+
+| COVERAGE     |                 | OFF     | Enables or disables coverage                                           |
++--------------+                 +---------+------------------------------------------------------------------------+
+| SWIG_PYTHON  |                 | OFF     | Enables or disables libraries and native interface bindings for python |
++--------------+                 +---------+------------------------------------------------------------------------+
+| SWIG_JAVA    |                 | OFF     | Enables or disables libraries and native interface bindings for java   |
++--------------+-----------------+---------+------------------------------------------------------------------------+
+
+Packaging Specific Parameters
+"""""""""""""""""""""""""""""
+
++-----------------------+-----------------+---------+--------------------------------------------+
+| Parameter             | Possible values | Default | Description                                |
++=======================+=================+=========+============================================+
+| ENABLE_LIBS_PACKAGING |      ON/OFF     | ON      | Enables or disables all types of packaging |
++-----------------------+                 +---------+--------------------------------------------+
+| PACKAGE_ZIP           |                 | OFF     | Enables or disables zip packaging          |
++-----------------------+                 +---------+--------------------------------------------+
+| PACKAGE_TGZ           |                 | OFF     | Enables or disables tar.gz packaging       |
++-----------------------+                 +---------+--------------------------------------------+
+| PACKAGE_RPM           |                 | OFF     | Enables or disables rpm packaging          |
++-----------------------+                 +---------+--------------------------------------------+
+| PACKAGE_DEB           |                 | OFF     | Enables or disables deb packaging          |
++-----------------------+-----------------+---------+--------------------------------------------+
+
+Running tests (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+After building Iroha, it is a good idea to run tests to check the operability
+of the daemon. You can run tests with this code:
+
+.. code-block:: shell
+
+  cmake --build build --target test
+
+Alternatively, you can run following command in the ``build`` folder
+
+.. code-block:: shell
+
+  cd build
+  ctest .
```

Comment:
`--output-on-failure` flag might be useful for us (e.g asking logs on issues or so)

Reply:
good point